gulp build problems on fresh install

There are a couple `Cannot find module` messages during `gulp build`. It looks like a couple files are buried in directories, but maybe links to them in code weren't change.
``` bash
[11:04:24] Starting 'browserify-publish'...
[11:04:24]
[11:04:24] Error: Cannot find module '../locale' from '/Users/secretrobotron/src/webmaker-app/publish'
    at /usr/local/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:43:25
    at load (/usr/local/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:99:43)
    at /usr/local/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:105:22
    at /usr/local/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at Object.oncomplete (fs.js:107:15)

[11:04:24] Finished 'browserify-publish' after 589 ms
[11:04:24]
[11:04:24] Error: Cannot find module '../locale' from '/Users/secretrobotron/src/webmaker-app/lib'
    at /usr/local/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:43:25
    at load (/usr/local/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:99:43)
    at /usr/local/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:105:22
    at /usr/local/lib/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at Object.oncomplete (fs.js:107:15)

[11:04:24] Finished 'browserify' after 663 ms
[11:04:24] Starting 'cache'...
[11:04:25] Finished 'cache' after 229 ms
[11:04:25] Finished 'build' after 3.27 s
```

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -29,7 +29,7 @@ gulp.task('build-locales', function (done) {
 gulp.task('template', ['config'], template);
 gulp.task('publish-assets', publish);
 gulp.task('build-static', function (done) {
-    sequence('clean', 'config', ['download-locales', 'publish-assets', 'template'], done);
+    sequence('clean', 'config', ['build-locales', 'publish-assets', 'template'], done);
 });
 
 // Browserify

